### PR TITLE
feat(charts): Add World Map "chart"

### PR DESCRIPTION
**Pull request by billyvg**
This adds a world map "chart" and a mapping of country codes to country name (for eCharts)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/charts/worldMapChart.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import BaseChart from './baseChart';
+import MapSeries from './series/mapSeries';
+
+export default class WorldMapChart extends React.Component {
+  static propTypes = {
+    ...BaseChart.propTypes,
+    seriesOptions: PropTypes.object,
+  };
+  constructor(props) {
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/charts/worldMapChart.jsx]: nit: space before constructor

File: src/sentry/static/sentry/app/components/charts/worldMapChart.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    // eslint-disable-next-line
+    this.setState({
+      countryCodesMap: countryCodesMap.default,
+    });
+  }
+
+  render() {
+    if (this.state.countryCodesMap === null) {
+      return null;
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/charts/worldMapChart.jsx]: Loading indicator?

**Review by evanpurkhiser — APPROVED** [review]
Looks great

File: src/sentry/static/sentry/app/components/charts/worldMapChart.jsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/charts/worldMapChart.jsx]: Going to hold off on this for now as requirements may change for this when we start to use it.

**Pull request merged**